src/jni/jvm_loader.cpp

```diff
@@ -69,7 +67,7 @@ String jni::JvmLoader::get_jvm_lib_path() {
 }
 
 String jni::JvmLoader::get_path_to_locally_installed_jvm() {
-    String javaHome{getenv("JAVA_HOME")};
+    String javaHome{OS::get_singleton()->get_environment("JAVA_HOME")};
 
     if (javaHome.empty()) {
         LOG_ERROR("JAVA_HOME is not defined! Exiting...")
```

Comment:
@chippmann Outside of this PR, but this probably causes the crash described in #133. Do we need JAVA_HOME to be always defined? or can we used the embedded jre?

Reply:
Yes this is definitely the issue. See the exit(1). we did this on purpose. IIRC we discussed this once and decided that during development we should use the locally installed jdk. But i guess we could also change that.

Reply:
Ahh, I guess that's fine as long as it is mentioned in the docs.